src/mudlet.qrc

```diff
@@ -169,6 +169,7 @@
         <file>icons/Slothmud.png</file>
         <file>icons/solid-line.png</file>
         <file>icons/stickmud_icon.jpg</file>
+        <file>icons/system-close.png</file>
```

Comment:
I'd be very reluctant to call this icon "system-close" as it is shutting a single profile down NOT the whole application... :worried:

Reply:
Please go with close-profile or profile-close if you will